pom.xml

```diff
@@ -89,9 +89,11 @@
         <jpro-auth-core.version>${jpro-platform.version}-ikmdev</jpro-auth-core.version>
 
         <!-- TestFX Variables -->
+        <tinkar-starter-data-reasoned.version>1.0.0</tinkar-starter-data-reasoned.version>
         <testfx.version>4.0.18-jpms</testfx.version>
         <openjfx-monocle.version>21.0.2-jpms</openjfx-monocle.version>
         <maven-surefire-plugin.version>3.5.0</maven-surefire-plugin.version>
```

Comment:
we need to get the parent pom to be updated to use this version of surefire/failsafe rather than changing this for individual projects.  We want to stay consistent on build tooling

Reply:
Correct! Now that the `java-parent` has been updated to the latest version, they are no longer needed here.